tests/unit/other.tcl

```diff
@@ -459,15 +464,35 @@ start_cluster 1 0 {tags {"other external:skip cluster slow"}} {
     } {} {needs:debug}
 
     test "Redis can rewind and trigger smaller slot resizing" {
+        # hashslot(foo) is 12182
         # hashslot(alice) is 749, smaller than hashslot(foo),
         # attempt to trigger a resize on it, see details in #12802.
         for {set j 1} {$j <= 128} {incr j} {
             r set "{alice}$j" a
         }
+
+        # disable resizing
+        r config set rdb-key-save-delay 10000000
+        r bgsave
+        wait_for_condition 1000 10 {
+            [s rdb_bgsave_in_progress] eq 1
+        } else {
+            fail "bgsave did not start in time"
+        }
```

Comment:
maybe instead we should wait for the previous bgsave to complete, before triggering the new one? (maybe this belongs in the previous test)?

Reply:
in line 453, we will wait until the previous bgsave to end.

Reply:
Using bgsave here makes sense.

Reply:
i now notice that the previous test ends with verifying that the bgsave is over. which means that when this test calls BGSAVE, it'll start instantly and there's also no need for this wait_for loop